jsk_topic_tools/src/lightweight_throttle_nodelet.cpp

```diff
@@ -94,13 +105,21 @@ namespace jsk_topic_tools
       advertised_ = true;
       pub_ = pnh_.advertise(opts);
     }
+
     ros::Time now = ros::Time::now();
-    if ((now - latest_stamp_).toSec() > 1 / update_rate_) {
+
+    if (latest_stamp_ > now) {
+      ROS_WARN("Detected jump back in time.");
```

Comment:
Hmm, I disagree.
Because
- with `Detected jump back in time`, I expect it does not change behavior. but it actually overwrites the last published time and it changes the publishing rate at that time.
- This node is for core developers IMO, and I'm not sure why you need to close the detailed behavior.

> users should not be conscious about latest_stamp_ variable

- You can add better message, for example `Detected jump back in time. Clearing time buffer of latest message publication (for throttling)`.

@k-okada What do you think?